### ISSUE TITLE
Support stateless functional components as a route action

### ISSUE THREAD

**Issue by kuy** [kuy/redux-saga-tower#9]
The route definition accepts only Component or PureComponent because it uses `isReactComponent` to determine components or generators.

**Comment by rpunkfu** [comment]
Is there any way to do so, expect checking if passed value is a function? I couldn't find any official method for it :)

**Comment by kuy** [comment]
Yes, so I need to call a function and check a return value. If a function is a stateless functional component, it may return ReactElement. If not so, it's a saga. Anyway, we can execute it using `call` effect.

**Comment by kuy** [comment]
I have to prepare automated test for first release.

**Comment by rpunkfu** [comment]
Yup, tests would be cool, if you manage to choose & setup tools for testing, I would be happy to help in writing them :)

**Comment by kuy** [comment]
I noticed that I can also use `is-generator` module to determine the value is a generator function or a normal function. `typeof value.prototype.isReactComponent === 'undefined' && !isGeneratorFunction(value)` is a stateless functional component.
